src/common/concurrent/entry_info.rs

```diff
@@ -1,19 +1,22 @@
-use std::sync::atomic::{AtomicBool, AtomicU32, Ordering};
+use std::sync::atomic::{self, AtomicBool, AtomicU16, AtomicU32, Ordering};
 
 use super::{AccessTime, KeyHash};
 use crate::common::{concurrent::atomic_time::AtomicInstant, time::Instant};
 
 #[derive(Debug)]
 pub(crate) struct EntryInfo<K> {
     key_hash: KeyHash<K>,
-    /// `is_admitted` indicates that the entry has been admitted to the
-    /// cache. When `false`, it means the entry is _temporary_ admitted to
-    /// the cache or evicted from the cache (so it should not have LRU nodes).
+    /// `is_admitted` indicates that the entry has been admitted to the cache. When
+    /// `false`, it means the entry is _temporary_ admitted to the cache or evicted
+    /// from the cache (so it should not have LRU nodes).
     is_admitted: AtomicBool,
-    /// `is_dirty` indicates that the entry has been inserted (or updated)
-    /// in the hash table, but the history of the insertion has not yet
-    /// been applied to the LRU deques and LFU estimator.
-    is_dirty: AtomicBool,
+    /// `entry_gen` (entry generation) is incremented every time the entry is updated
+    /// in the concurrent hash table.
+    entry_gen: AtomicU16,
+    /// `policy_gen` (policy generation) is incremented every time entry's `WriteOpe`
```

Comment:
```suggestion
    /// `policy_gen` (policy generation) is incremented every time entry's `WriteOp`
```

Will fix in a separate PR #363.

Reply:
Fixed the typo by this commit for this PR:

- https://github.com/moka-rs/moka/pull/348/commits/b866dce0e9343d17bb808c9b35c87503fbbe28c0#diff-96c4a5be5f17869907f65e314ff6a08126fb0aef7489359b9ebdfcf2984ee399R16